shuup/admin/templates/shuup/admin/products/edit.jinja

```diff
@@ -19,6 +19,26 @@
                     </ul>
                 </div>
             {% endif %}
+            {% if validation_issues %}
+                {# TODO: FIX THIS FOR MOBILE SCREEN SIZES #}
+                <div class="validation-issues-alert alert alert-{{ validation_issues[0].get_alert_type() }}"
```

Comment:
every issue should be a alert div, otherwise you can group errors with info:

```jinja
{% for issue in validation_issues %}
    {% set issue_content = bleach.clean(issue.message|string, strip=True) %}
    <div class="validation-issues-alert alert alert-{{ issue.get_alert_type() }}" data-code="{{ issue.code }}" data-title="{{ issue_content }}">
      {{ issue_content|safe if issue.is_html else issue_content }}
    </div>
{% endfor %}
```

Make sure to use strip=True to strip not allowed tags